tests/specs/file.js

```diff
@@ -65,20 +65,71 @@ define(["dist/resolver"], function(Resolver) {
     });
 
 
-    describe("when addExtension", function() {
-      it("then `test/file.html` is `test/file.html", function() {
-        var fileString = File.addExtension("test/file.html", "badextension");
-        expect(fileString).to.equal("test/file.html");
+    describe("addExtension", function() {
```

Comment:
This was the only describe under "File" which used to start with when, so that is why I changed it. I will change it back now and submit a PR later changing all of them to "when calling ____" if that is agreeable to you.